### ISSUE TITLE
Remain public or go private?

### ISSUE THREAD

**Issue by lzachmann** [csp-inc/aesthetics#3]
Determine whether we should make this a private repository, at least until we feel comfortable enough with the contents to make it more public. Note that the GitHub Page for the project is always public, even if the repo is private.

**Comment by brettgdickson** [comment]
For now, this repository should remain private, and available (for now) only to the 'aesthetics team.' For example, we have not actually adopted some of these ideas (e.g., font style), so this should be a place for exchanging ideas first.

**Comment by lzachmann** [comment]
Agreed, thanks Brett. I'll work with Mary to set up payment for the [Bronze plan](https://github.com/pricing). It's $25 a month and will allow us to host up to 10 private repositories.

**Comment by mobeaner** [comment]
Sounds good to me, too.
